HACKING.md

```diff
@@ -10,38 +10,42 @@ make things easier...
    `sdn-${OPENSHIFT_RELEASE_NUMBER}-kubernetes-${KUBERNETES_RELEASE_NUMBER}`,
    based on the latest upstream release/pre-release tag. eg, the first
    rebase attempt for OCP 4.3 was the branch
-   `sdn-4.3-kubernetes-1.16.0-beta.2`, based off the upstream tag
-   `v1.16.0-beta.2`. The `${OPENSHIFT_RELEASE_NUMBER}` should just be
+   `sdn-4.4-kubernetes-1.17.2`, based off the upstream tag
+   `v1.17.2`. The `${OPENSHIFT_RELEASE_NUMBER}` should just be
    `MAJOR.MINOR`, but the `${KUBERNETES_RELEASE_NUMBER}` should be
    exactly the name of the upstream tag you used, minus the initial
-   "`v`".
+   "`v`". Sometimes https://github.com/openshift/kubernetes won't have all the
+   tags, make sure to get the latest tag from
+   https://github.com/kubernetes/kubernetes 
 
 3. Peruse the previous release's branch using your favorite git tool.
    If you're not sure what branch that is, look in [sdn's
-   `glide.yaml`](./glide.yaml):
+   `go.mod`](./go.mod):
 
        ...
-       - package: k8s.io/kubernetes
-         repo:    https://github.com/openshift/kubernetes.git
-         version: sdn-4.2-kubernetes-1.14.0
+         [...]
+         replace (
+            [...]
+            k8s.io/kubernetes => github.com/openshift/kubernetes v1.17.0-alpha.0.0.20190924141618-7eb200efda20
+            [...]
+         )
        ...
 
+   Unfortunately, go mod will autogenerate the version in a non human-friendly
+   way, in this case, `7eb200efda20` is the commit number. One way to see the
+   branch is going to the github.com/openshift/kubernetes repository, checkout
+   `7eb200efda20` and the use git log --oneline --decorate
+
    So, eg, [that branch in
-   `openshift/kubernetes`](https://github.com/openshift/kubernetes/commits/sdn-4.2-kubernetes-1.14.0)
+   [`openshift/kubernetes`](https://github.com/openshift/kubernetes/commits/sdn-4.3-kubernetes-1.16.0)
```

Comment:
There's already an open bracket on the previous line. Don't add a second one here.